src/pages/comparison/comparison.tsx

```diff
@@ -537,15 +678,24 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
           <RightLayersPanelWrapper layout={layout}>
             <LayersPanel
               id="right-layers-panel"
-              onMapsSelect={onMapsSelect}
-              onLayersSelect={(layers: LayerExample[]) =>
-                setLayerRightSide(layers[0])
+              layers={rightSideExamples}
+              selectedLayerIds={[rightSideSelectedExampleId]}
+              onLayerInsert={(newLayer) =>
+                handleInsertExample(newLayer, "right")
               }
+              onMapsSelect={onMapsSelect}
+              onLayerSelect={(id: string) => handleSelectExample(id, "right")}
+              onLayerDelete={(id) => handleDeleteExample(id, "right")}
               onPointToLayer={() => onPointToLayer("right")}
               type={ListItemType.Radio}
               onClose={() =>
                 handleChangeRightPanelVisibility(ActiveButton.options)
               }
+              baseMaps={baseMaps}
+              selectedBaseMapId={selectedBaseMapId}
+              onBaseMapInsert={handleInsertBaseMap}
+              onBaseMapSelect={handleSelectBaseMap}
```

Comment:
`onMapSelect` and `onBaseMapSelect` look weird.

Reply:
Renamed